data/constants/keycodes_0.0.1.json

```diff
@@ -0,0 +1,93 @@
+{
+    "ranges": {
+        "0x0000/0x00FF": {
+            "define": "QK_BASIC"
+        },
+        "0x0100/0x1EFF": {
+            "define": "QK_MODS"
+        },
+        "0x2000/0x1FFF": {
+            "define": "QK_MOD_TAP"
+        },
+        "0x4000/0x0FFF": {
+            "define": "QK_LAYER_TAP"
+        },
+
+        "0x5000/0x001F": {
+            "define": "QK_TO"
+        },
+        "0x5020/0x001F": {
+            "define": "QK_MOMENTARY"
+        },
+        "0x5040/0x001F": {
+            "define": "QK_DEF_LAYER"
+        },
+        "0x5060/0x001F": {
+            "define": "QK_TOGGLE_LAYER"
+        },
+        "0x5080/0x001F": {
+            "define": "QK_ONE_SHOT_LAYER"
+        },
+        "0x50A0/0x001F": {
+            "define": "QK_LAYER_TAP_TOGGLE"
+        },
+
+
+        "0x5100/0x00FF": {
+            "define": "QK_ONE_SHOT_MOD"
+        },
+        "0x5200/0x00FF": {
+            "define": "QK_LAYER_MOD"
+        },
```

Comment:
Maybe extend the size of this block to `0x01FF`, so that right side modifiers could be supported? Apparently some people really want that (#13085).